wandb/sdk/data_types/object_3d.py

```diff
@@ -198,10 +198,12 @@ def __init__(
 
             if len(np_data.shape) != 2 or np_data.shape[1] not in {3, 4, 6}:
                 raise ValueError(
-                    """The shape of the numpy array must be one of either
+                    """
+                    The shape of the numpy array must be one of either
                                     [[x y z],       ...] nx3
                                      [x y z c],     ...] nx4 where c is a category with supported range [1, 14]
-                                     [x y z r g b], ...] nx4 where is rgb is color"""
+                                     [x y z r g b], ...] nx4 where rgb is color
```

Comment:
is it actually nx4 should it be nx6?

Also it seems misleading that we have different length in the same array?